Source/common/SNTConfigurator.m

```diff
@@ -1609,6 +1609,13 @@ - (nullable NSArray *)validateConfiguration {
     CFStringRef keyRef = (__bridge CFStringRef)key;
     if (!CFPreferencesAppValueIsForced(keyRef, kMobileConfigDomain)) return;
 
+    // If the key is a 'standard' configuration profile key, skip it.
```

Comment:
Why are these nested in the Santa config?

Reply:
See the changes in https://github.com/northpolesec/santa/pull/487.

Reply:
I reckon I thought we were just inspecting the payload content section, but doctor is looking at the whole mobileconfig? If so, should there be other ignored keys as well?